lib/src/main/java/io/ably/lib/http/AsyncHttpScheduler.java

```diff
@@ -12,23 +9,32 @@
  * A HttpScheduler that uses a thread pool to run HTTP operations.
  */
 class AsyncHttpScheduler extends HttpScheduler<ThreadPoolExecutor> {
-	AsyncHttpScheduler(HttpCore httpCore, ClientOptions options) {
-		super(httpCore, new ThreadPoolExecutor(options.asyncHttpThreadpoolSize, options.asyncHttpThreadpoolSize, KEEP_ALIVE_TIME, TimeUnit.MILLISECONDS, new LinkedBlockingQueue<Runnable>()));
-		executor.allowsCoreThreadTimeOut();
-	}
+    AsyncHttpScheduler(HttpCore httpCore, ClientOptions options) {
```

Comment:
Why have you changed from tabs to spaces here?

Reply:
@paddybyers so Github renders the code with the correct indentation. For instance, look how [SyncHttpScheduler](https://github.com/ably/ably-java/blob/develop/lib/src/main/java/io/ably/lib/http/SyncHttpScheduler.java) is displayed now: the Github web view renders TAB as 8 spaces instead of 4.